crates/connectors/ndc-postgres/src/schema.rs

```diff
@@ -186,52 +186,59 @@ pub async fn get_schema(
         BTreeMap::from_iter(metadata.tables.0.iter().map(|(collection_name, table)| {
```

Comment:
FWIW, `BTreeMap::from_iter` might be more idiomatically expressed as `.collect()`.

Reply:
good to know!